lib/helpers.js

```diff
@@ -90,6 +91,28 @@ function getIMEListFromOutput (stdout) {
   return engines;
 }
 
+async function getSdkToolsVersion() {
+    return new Promise((resolve) => {
+        const androidHome = process.env.ANDROID_HOME;
+        let tools = path.resolve(androidHome, 'tools');
+
+        var rd = readline.createInterface({
+            input: fs.createReadStream(tools + '/source.properties'),
```

Comment:
This will break on Windows, with `/` as the separator. This is what `path` is for.